plexmuxy/logging_utils.py

```
@@ -0,0 +1,22 @@
+
+import logging
+from datetime import datetime
+from pathlib import Path
+
+from .config import platform_config_path
+
+
+def configure_logging(verbose: bool = False) -> Path:
+    log_dir = platform_config_path().parent / "logs"
+    log_dir.mkdir(parents=True, exist_ok=True)
+    log_path = log_dir / f"{datetime.now().strftime('%Y%m%d-%H%M%S')}.log"
```

Ruff (0.15.18)

[warning] `datetime.datetime.now()` called without a `tz` argument

(DTZ005)